server/app.js

```diff
@@ -1,21 +1,21 @@
 import express from 'express';
 import bodyParser from 'body-parser';
+// import user from './routes/user';
 import index from './db';
-import user from './routes/user';
+import auth from './routes/auth';
 import request from './routes/request';
 
+require('dotenv').config();
 
 const app = express();
-const port = 9000;
+const port = 3000;
 app.use(bodyParser.urlencoded({ extended: true }));
 app.use(bodyParser.json());
 
+app.use('/auth', auth);
 app.use('/', request);
 // app.use('/users', user);
-
-
-app.listen(port, () => {
-  console.log(`app started at localost ${port}`);
-});
+if (!module.parent) { app.listen(port); }
```

Comment:
what is the` module.parent` for?

Reply:
It due to an error, I had with my node, "Error: EARNINUSE",  I have removed it.